src/base-command.js

```diff
@@ -33,6 +34,18 @@ function aioManifestToOpenwhiskAction(manifestAction) {
     return owAction;
 }
 
+async function execute(command, args) {
+    const result = spawnSync(command, args, {stdio: "inherit"});
```

Comment:
`spawnSync()` essentially equal to `await exec()`?

Reply:
Yeah, not the same. I mostly used it here to get streaming output to the console – together with `stdio: "inherit"` setting – as `exec()/execSync()` are geared to give you the stdout as return value. And I wanted error handling.